assets/static/templates.js

```diff
@@ -0,0 +1,67 @@
+var Templates = (function(params) {
+
+
+    var templates = {},
+        config = {
+          // popver with the list of most common labels
```

Comment:
Also, the indentation here is a bit wonky. It seems like the rest of the file uses 4 spaces but `config` only use 2.